examples/llm/train_clm/yamls/hf_causal_lm/gpt-neo-125m.yaml

```diff
@@ -85,11 +85,10 @@ precision: amp_bf16
 # FSDP
```

Comment:
Perhaps outside the scope of this PR, but is it actually recommended to use FSDP on such lightweight models?

If no, then I might recommend that, where appropriate, we comment out the `fsdp_config` blocks and leave a comment saying something along the lines of "you can comment/uncomment this block to turn FSDP off/on; it is typically not required for the workload in this config".

Reply:
I think.. its probably not necessary, but it doesn't hurt... if someone is trying this on some low resource cards at home, maybe it still help them out.

I will make a JIRA for this but not address in this PR